docs/telemetry.md

```diff
@@ -1,6 +1,15 @@
 # Telemetry
 For clients that have "Send usage data" enabled, Firefox for Echo Show sends a "core" ping and an "event" ping to Mozilla's telemetry service. Sending telemetry can be disabled in the app's settings. Builds of Firefox for Echo Show have telemetry enabled by default ("opt-out").
 
+## Glean telemetry
+Glean telemetry documentation [here](https://mozilla.github.io/glean/book/user/pings/index.html).
+
+### Baseline ping
+Firefox for Echo Show creates and tries to send a "baseline" ping when the app goes to the background. This baseline ping is defined by the [Glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean) component and [documented in the Android Components repository](https://mozilla.github.io/glean/book/user/pings/baseline.html).
+
+### Metrics ping
+The default ping for metrics. Sent approximately daily. https://github.com/mozilla-mobile/firefox-echo-show/tree/master/docs/metrics.md
+
```

Comment:
As per my previous comment, this is bound to become obsolete/outdated. Please just link to the `app/docs/metrics.md` file here: if you ever add metrics/pings, or if the SDK add pings, this won't show them unless you manually add it. But the autogenerated docs will do so.

Reply:
For example, this is my suggested change for Fenix as well mozilla-mobile/fenix#12421